evaluations/evaluation_aragog.py

```diff
@@ -153,15 +160,20 @@ def create_args():
 
 
 def main():
+    t1 = datetime.datetime.now()
     args = create_args()
     questions, answers = read_question_answers()
 
     if args.sample:
         random.seed(42)
-        questions = random.sample(questions, args.sample)
-        answers = random.sample(answers, args.sample)
+        sampled_ids = random.sample(range(len(questions)), args.sample)
```

Comment:
Thanks! This was indeed a bug, but this does not affect the whole results, since they were generated without any sampling.